query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/QueryBuilderDsl.java

```diff
@@ -0,0 +1,742 @@
+    return getFunction(CqlIdentifier.fromCql(functionName), arguments);
+  }
+
+  /** Var-arg equivalent of {@link #getFunction(String, Iterable)}. */
+  static Selector getFunction(String functionName, Selector... arguments) {
+    return getFunction(functionName, Arrays.asList(arguments));
+  }
+
+  /**
+   * Selects the result of a function call, as is {@code SELECT ks.f(a,b)}
+   *
+   * <p>None of the arguments should be aliased (the query builder checks this at runtime).
+   *
+   * @throws IllegalArgumentException if any of the selectors is aliased.
+   */
+  static Selector getFunction(
+      CqlIdentifier keyspaceId, CqlIdentifier functionId, Iterable<Selector> arguments) {
+    return new FunctionSelector(keyspaceId, functionId, arguments);
+  }
+
+  /** Var-arg equivalent of {@link #getFunction(CqlIdentifier, CqlIdentifier, Iterable)}. */
+  static Selector getFunction(
+      CqlIdentifier keyspaceId, CqlIdentifier functionId, Selector... arguments) {
+    return getFunction(keyspaceId, functionId, Arrays.asList(arguments));
+  }
+
+  /**
+   * Shortcut for {@link #getFunction(CqlIdentifier, CqlIdentifier, Iterable)}
+   * getFunction(CqlIdentifier.fromCql(functionName), arguments)}.
+   */
+  static Selector getFunction(
+      String keyspaceName, String functionName, Iterable<Selector> arguments) {
+    return getFunction(
+        CqlIdentifier.fromCql(keyspaceName), CqlIdentifier.fromCql(functionName), arguments);
+  }
+
+  /** Var-arg equivalent of {@link #getFunction(String, String, Iterable)}. */
+  static Selector getFunction(String keyspaceName, String functionName, Selector... arguments) {
+    return getFunction(keyspaceName, functionName, Arrays.asList(arguments));
+  }
+
+  /**
+   * Shortcut to select the result of the built-in {@code writetime} function, as in {@code SELECT
+   * writetime(c)}.
+   */
+  static Selector getWriteTime(CqlIdentifier columnId) {
+    return getFunction(WRITETIME, getColumn(columnId));
+  }
+
+  /**
+   * Shortcut for {@link #getWriteTime(CqlIdentifier)
+   * getWriteTime(CqlIdentifier.fromCql(columnName))}.
+   */
+  static Selector getWriteTime(String columnName) {
+    return getWriteTime(CqlIdentifier.fromCql(columnName));
+  }
+
+  /**
+   * Shortcut to select the result of the built-in {@code ttl} function, as in {@code SELECT
+   * ttl(c)}.
+   */
+  static Selector getTtl(CqlIdentifier columnId) {
+    return getFunction(TTL, getColumn(columnId));
+  }
+
+  /** Shortcut for {@link #getTtl(CqlIdentifier) getTtl(CqlIdentifier.fromCql(columnName))}. */
+  static Selector getTtl(String columnName) {
+    return getTtl(CqlIdentifier.fromCql(columnName));
+  }
+
+  /**
+   * Builds a relation testing a column.
+   *
+   * <p>This call must be chained with an operator, for example:
+   *
+   * <pre>{@code
+   * selectFrom("foo").getAll().where(isColumn("k").eq(bindMarker()));
+   * }</pre>
+   */
+  static ColumnRelationBuilder isColumn(CqlIdentifier id) {
```

Comment:
`isColumn` seems overly descriptive.  Wouldn't `column` be adequate?  i.e.

```java
selectFrom("foo").all().where(column("k").notNull());
```

seems less natural than:

```java
selectFrom("foo").all().where(column("k").notNull());
```

Same would apply for other `isXXX` methods.

Reply:
The methods are prefixed to avoid name collisions in `QueryBuilderDsl`. See my other reply for why we need `getXxx`.